llm/qwen/serve-110b.yaml

```diff
@@ -0,0 +1,43 @@
+envs:
+  MODEL_NAME: Qwen/Qwen1.5-110B-Chat
+
+service:
+  # Specifying the path to the endpoint to check the readiness of the replicas.
+  readiness_probe:
+    path: /v1/chat/completions
+    post_data:
+      model: $MODEL_NAME
+      messages:
+        - role: user
+          content: Hello! What is your name?
+      max_tokens: 1
+    initial_delay_seconds: 1200
+  # How many replicas to manage.
+  replicas: 2
+  
+
+resources:
+  accelerators: {A100-80GB:4, A100-80GB:8}
```

Comment:
If it works with `A100-80GB:4`, it should probably work with `A100:8` as well : )

Reply:
Ahh I could not get availability on A100:8 so couldn't test, so I didn't include it. Added it now.